docs/glossary.md

```diff
@@ -56,6 +56,48 @@ It is designed to bring autoscaling to your production environment, meaning it c
 
 A RunPod template is a Docker container image paired with a configuration.
 
+## Total Workers
+
+Total workers refer to the total number of workers available to your account, which is the sum of the max workers across all your endpoints. If you run out of total workers, please reach out to us by [creating a support ticket](https://contact.runpod.io/).
+
+## Initial Workers
+
+When you first create your endpoint, RunPod needs to download the Docker image for your workers, and they will be marked as initializing during this process.
+
+## Active (Min) Workers
+
+“Always on” workers. Setting active workers to 1 or more ensures that a worker is always ready to respond to job requests without cold start delays.
+
+Default: 0
+
+:::note
+
+You will be charged for all active workers. However, when active workers are not processing jobs, they will receive up to a 30% discount on the regular cost.
```

Comment:
This is incorrect. The up to 30% discount is for active workers in general, and there's no difference in rate based on whether they're processing a job or not.